pp_network.py

```diff
@@ -17,12 +17,15 @@ class NetModel(object):
     
     def __init__(self, net_given=None, network_name='rural_1',
                  zero_out_gen_shunt_storage=False, tstep=1./60,
-                 net_zero_reward=1.0, env_name=None, baseline=True):
+                 net_zero_reward=1.0, env_name=None, baseline=True, config=None):
         """Initialize attributes of the object and zero out certain components
         in the standard test network."""
 
         if env_name is not None:
-            self.config = get_config(env_name, baseline)
+            if config is not None:
```

Comment:
it's redundant to have the user give both an environment name and a config file. If the user wants to pass a custom config file for which there isn't a name assigned yet, this code won't let them do that. So, let's change the logic to check if a config object is passed first (if so, use it), and if there isn't one, then try to load one using `get_config(env_name, baseline)`.

Reply:
Changed it, good catch!